src/test/scala/org/camunda/feel/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -205,6 +205,8 @@ class BuiltinNumberFunctionsTest
 
     eval(" odd(5) ") should be(ValBoolean(true))
     eval(" odd(2) ") should be(ValBoolean(false))
+    eval(" odd(-5)") should be(ValBoolean(true))
+    eval(" odd(-2)") should be(ValBoolean(false))
```

Comment:
In order to separate the cases, we can create a new test function below. For example:

``` 
it should "return true if a negative number is odd" in {
    eval(" odd(-5)") should be(ValBoolean(true))
    eval(" odd(-2)") should be(ValBoolean(false))
}
```

Reply:
Separated code into its own test.